src/components/reactHooks/context.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, createContext } from "react";
+import Footer from "../leyout/footer";
+import Header from "../leyout/header";
+import ContextChildren from "../../pages/contextChildren";
+
+export const UserNameContext = createContext();
+
+const Context = () => {
+  const [inputHandler, setInputHandler] = useState();
+  const [name, setName] = useState("React Context");
+
+  const hendlerInput = (event) => {
```

Comment:
typo: should be "handler"
```suggestion
  const handlerInput = (event) => {
```